Develop

mikethreels opened the pull request:
**The new web responsive design**

Nav
Included the logo and disappearing elements depending on the screen size
The logo is responsive and changes position on resolutions higher than 1024.

Main
The sections are responsive and display next to each other 768px using bootstrap.
Fluid containers to adapt to the screen size between the breakpoints

Responsive
Responsive designt with 3 breakpoints
480px
768px
1024px

Misc
-Included original fonts
-Added background pictures
-Replaced SVG icons with font awesome icons

lucasmazo32 commented on index.html:
```diff
@@ -0,0 +1,1356 @@
+                </ul>
+            </nav>
+        </header>
+        <main class="container">
+            <div class="row">
+                <section class="top-story-section col-sm-12 col-md-7 order-md-2 col-lg-5 order-lg-2">
+                    <div class="sticky">
+                        <div class="section-title">
+                            <h2><a href="#">Top story</a></h2>
+                        </div>
+                        <article class="story ">
+                            <div class="story-img-container">
+                                <div class="story-img-category">
+                                    <a href="#">u.s.</a>
+                                </div>
+                                <div class="story-img">
+                                    <a href="#">
+                                        <img src="images/mcconnell-stimulus.jpg" alt="crowded street">
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="story-content">
+                                <h2><a href="#">New Stimulus Could Come in 'Next Month or So,' McConnell Says</a></h2>
+                                <div class="summary">
+                                    <p>The Kentucky Republican has said in recent weeks that he is eyeing another stimulus worth around $1 trillion that likely will not include another round of checks for Americans.</p>
+                                </div>
+                            </div>
+                        </article>
+                    </div>
```
- [ ] I am getting more margin than expected in here. Kindly fix that. I am getting the same issue for small screens. Instead of having a container, I think you can have a container fluid and set a max-width. I think that's the way they achieve it. You can check that!

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83210784-5b8a3b80-a121-11ea-9660-6cc2d350c984.png) | ![image](https://user-images.githubusercontent.com/57414789/83210751-457c7b00-a121-11ea-923a-7434e3fa07cd.png)

lucasmazo32 commented on index.html:
```diff
@@ -0,0 +1,1356 @@
+                    </article>
+                </div>
+            </section>
+            <div class="last-bottom-pick-section">
+                <div class="last-bottom-pick">
+                  <div class="block-title no-border">
+                    <a href="#">U.S.</a>
+                  </div>
+                  <div class="row">
+                    <div class="col-md-4">
+                      <article>
+                        <div class="article-wrap">
+                          <div class="featured-article-img url1"></div>
+                        </div>
+                        <h4>
+                          <a href="#">
+                            Why a Bipartisan Revolt Killed the House's Spy Bill
+                          </a>
+                        </h4>
+                      </article>
+                    </div>
+                    <div class="col-md-4">
+                      <article>
```
- [ ] For some reason this section is not displayed for medium or small devices. Kindly check that.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83211318-68f3f580-a122-11ea-8769-1bd2f29eeafb.png) | ![image](https://user-images.githubusercontent.com/57414789/83211286-54aff880-a122-11ea-94ab-5ad32c781335.png)

mikethreels commented:
**changes made**
Thanks a lot for the tip with the container fluid I was actually struggling with getting that right.
the adjustments are made as requested.
- added "container-fluid" instead of "container".
- added the final section to the smaller break-points as well.

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

mikethreels commented:
**Update**
as you can see in the below print screen from my live demo the responsive feature is still working in the same screen size you're showing.
therefore I'm not sure what to adjust in this case, would it be possible to elaborate on the issue?
![tempsnip](https://user-images.githubusercontent.com/39240820/83313340-0d0a9900-a216-11ea-9d2c-e9a7dd088737.png)

trekab commented on styles/style.css:
```diff
@@ -0,0 +1,855 @@
+  text-transform: uppercase;
+}
+
+.subscribe-section {
+  flex-direction: column;
+}
```
This could be achieved with bootstrap. ([Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))

trekab commented on styles/style.css:
```diff
@@ -0,0 +1,855 @@
+  background-position: center;
+}
+
+.subscribe-img {
+  justify-content: center;
+}
```
This could also be achieved with bootstrap. ([Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))

trekab commented on styles/style.css:
```diff
@@ -0,0 +1,855 @@
+
+@media (max-width: 1023px) {
+  nav,
+  .weather,
+  .nav-subscribe {
+    display: none;
+  }
+}
+
+@media (min-width: 1023px) {
+  .navbar-logo svg {
+    fill: white;
+    width: 396px;
+    height: 60px;
+  }
+
+  nav {
+    margin-top: 90px;
+  }
+
+  .menu-button,
+  .free-trial-button {
+    display: none;
+  }
+
+  .sign-up-container {
+    margin-top: 35px;
+  }
+
+  .subscribe-title {
+    max-width: 250px;
+  }
+}
+
+.last-bottom-pick-side li:first-child {
+  padding-top: 0;
+}
+
+/* helper */
+.flex { display: flex; }
```
This could also be refined further by relying on bootstrap classes. ([Responsive-breakpoints](https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints))

mikethreels commented:
### Changes made as suggested

- changed the login icon to a login link in the 2 biggest breakpoints
- decreased the amount of space between the navbar and the articles in the 2 smallest breakpoints
- replaced flex and grid with bootstrap in the media queries
